website/allowed-subcategories.txt

```diff
@@ -67,7 +67,7 @@ Lambda
 License Manager
 Lightsail
 MQ
-Macie
+Macie Classic
```

Comment:
Can you update the terraform.io documentation sidebar in `website/aws.erb` as well? Thanks!

Reply:
Done.